### ISSUE TITLE
Redirect logging to file

### ISSUE THREAD

**Issue by John-Gee** [zom-ponks/rpcs3-gameupdater#10]
With PR#9 we can have various levels from none to debug, but it's all to terminal, it may be good to also/instead redirect it to file.

**Comment by zom-ponks** [comment]
Yes, maybe do it like this: in command line operation, don't log to file (unless specified by a switch?), but in GUI mode do it automatically (unless somehow specified not to).

**Comment by John-Gee** [comment]
That looks good to me, yet I'm thinking it'd be more confusing for the users to have a different behavior depending on the interface chosen.
I'm conflicted whether logging to file should be on or not by default so I'll leave that to you.


Your could also have a panel in the GUI that displays all the logging (like RPCS3) which would be equivalent to logging to the terminal for the CLI. That'd be clean I think.

**Comment by zom-ponks** [comment]
+1 for Logging panel, maybe an option to save to file. Then have a --log <filename> switch on the CLI.

That way we could keep the behavior consistent between UIs.

**Comment by John-Gee** [comment]
Yup that sounds good.
